qiskit/chemistry/results/electronic_structure_result.py

```diff
@@ -67,31 +67,31 @@ def nuclear_dipole_moment(self, value: DipoleTuple) -> None:
     # construct the circuit of the GS from here (if the algorithm supports this)
 
     @property
-    def energy(self) -> Optional[float]:
```

Comment:
I made comments to this file in the Bosonic PR - the PRs should not be trying to push the same files or if they are then is there some order that these PRs need to be merged in if its changes upon changes which are still pending to be merged.

Reply:
again we need to first merge excited states then bosonic.